pom.xml

```diff
@@ -26,7 +26,7 @@
 		<assertj.version>3.4.1</assertj.version>
 		<mockito.version>1.10.19</mockito.version>
 		<tds-common.version>3.1.1.RELEASE</tds-common.version>
-		<item-renderer.version>3.1.2</item-renderer.version>
+		<item-renderer.version>3.1.5-SNAPSHOT</item-renderer.version>
```

Comment:
Will update this to non-snapshot version once ItemRenderer is ready to be released.